msal/build.gradle

```diff
@@ -79,6 +85,11 @@ android {
             def fileName = "${archivesBaseName}-${version}.aar"
             outputFileName = fileName
         }
+
+        def buildVariant = variant.name
+        createCoverageTask(buildVariant, "UnitTest")
+        createCoverageTask(buildVariant, "AndroidTest")
+        createCoverageTask(buildVariant, "BothTest")
```

Comment:
The Strings `UnitTest`, `AndroidTest` and `BothTest` can be stored in some constants as they are being reused.

Reply:
added